src/dune_engine/vcs.ml

```diff
@@ -67,7 +67,10 @@ let run t args =
   String.trim s
 
 let git_accept () =
-  Process.Accept (Predicate_lang.union [ Element 0; Element 128 ])
+  Process.Accept
+    (Predicate.create
+       ~id:(lazy (Dyn.Set [ Dyn.int 0; Dyn.int 128 ]))
+       ~f:(fun x -> Int.equal x 0 || Int.equal x 128))
```

Comment:
This diff doesn't seem great either. There is a lot of noise in the new expression.

Reply:
We could add a `Predicate.one_of` constructor to simplify this. Personally, I'd rather split "identifiable" and raw predicates as I mentioned in the PR description and the whole `~id` mess can disappear as it's useless here.

Reply:
:+1: